compiler/testData/codegen/boxInline/smap/anonymous/lambda.kt

```diff
@@ -1,5 +1,6 @@
-// FILE: 1.kt
 // IGNORE_BACKEND: JVM_IR
+// FILE: 1.kt
+
```

Comment:
New line causes smap data shift:  https://teamcity.jetbrains.com/viewLog.html?buildId=2450360&buildTypeId=Kotlin_dev_CompilerWithTests

Reply:
It's best to revert to original:
```
// FILE: 1.kt
// IGNORE_BACKEND: JVM_IR
package builders
``

Reply:
Done.